spi/src/main/java/org/arquillian/cube/spi/metadata/HasMappedPorts.java

```diff
@@ -0,0 +1,13 @@
+package org.arquillian.cube.spi.metadata;
+
+
+public interface HasMappedPorts extends CubeMetadata {
+    
+    String getIP();
+
+    Integer forTargetPort(int targetPort);
```

Comment:
I think for this issue we can do the following:

* Deprecate bindings()
* Implement a HasMappedPorts wrapper on top of bindings() for DockerCube so we can focus on HasMappedPorts as the single interface for deatling with what we're dealing with here
* Update HasMappedPorts to support IP per Port lookup.

Then we'll deal with updating the rest of Cube around binding later.